odoo/addons/base/views/ir_filters_views.xml

```diff
@@ -5,24 +5,42 @@
             <field name="model">ir.filters</field>
             <field name="arch" type="xml">
                 <form string="Filters">
-                  <sheet>
-                    <group col="4">
-                        <field name="name"/>
-                        <field name="user_id" string="Available for User"/>
-                        <field name="model_id"/>
-                        <field name="is_default"/>
-                        <field name="action_id"/>
-                        <field name="active" widget="boolean_toggle"/>
-                    </group>
-                    <group>
-                        <field name="domain" widget="domain" options="{'model': 'model_id'}"/>
-                        <field name="context"/>
-                        <field name="sort"/>
-                    </group>
-                   </sheet>
+                    <!-- required for the domain widget model option when the user is not in debug -->
+                    <field name="model_id" invisible="1"/>
```

Comment:
```suggestion
```suggestion
                    <field name="model_id" invisible="1"/>   <!-- required for the domain widget model option when the user is not in debug -->
```
To be parsed by the TestInvisibleField test
```